### ISSUE TITLE
Peculiar dark areas in Finland

### ISSUE THREAD

**Issue by jirikadlec2** [der-stefan/OpenTopoMap#327]
While browsing opentopomap.org in Finland, I have found some strange looking areas in dark green / gray colour, for example in:

**Comment by der-stefan** [comment]
Yeah, those are inaccuracies in the DEM. Those will be solved with the new DEM once we publish the new vector maps.
